stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/channel/add/AddChannelViewModel.kt

```diff
@@ -112,7 +114,7 @@ class AddChannelViewModel : ViewModel() {
             return
         }
         viewModelScope.launch(Dispatchers.IO) {
-            val currentUserId = chatDomain.user.value?.id ?: error("User must be set before create new channel!")
+            val currentUserId = chatClient.globalState.user.value?.id ?: error("User must be set before create new channel!")
```

Comment:
We can also call `chatClient.getCurrentUser()?.id`

Reply:
I would prefer to use globalState directly to request to user and added GlobalState as a dependency of this ViewModel, then we avoid accessing globalState thought ChatClient.